docs/framework/winforms/advanced/about-gdi-managed-code.md

```diff
@@ -12,7 +12,7 @@ ms.topic: "article"
 helpviewer_keywords: 
   - "GDI+, about GDI+"
```

Comment:
not related to your PR and not asking you to fix this, but having an entry GDI+ and another one GDI+, about GDI+ is not super helpful IMO. Thoughts?

Reply:
Agreed. I saw many things like this that I would fix if I had a few free days :-) Some topics have way too many keywords, some too few, and some keywords aren't really useful.

Reply:
Cool. We'll address this separately. Don't worry about this. You're already helping a lot @mikeblome!